stf.build/include/top.xml

```diff
@@ -46,7 +46,7 @@ limitations under the License.
 	</condition>
 	
 	<!--Specifry ASM version to be used-->
-	<property name="asm-version" value="7.3.1"/>
+	<property name="asm-version" value="9.0"/>
```

Comment:
Could we not include the version in the jar name? Instead of downloading into `asm-9.0/asm-9.0.jar`, just use a generic name. For example, `asm.jar`. If we do it this way, we do not need to update the folder and file name for every version update in https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/387 and https://github.com/AdoptOpenJDK/openjdk-tests/pull/2126

Reply:
The logic should be updated in https://github.com/AdoptOpenJDK/stf/blob/1f742d4a91b759d5181c66f15c6feaa7f4087ead/stf.build/include/top.xml#L346

Reply:
It could be. And that'd make the task of updating ASM library simpler. 

The downside of that would be that, we'd not be able to tell which version of asm jar we are using easily. Currently we can tell which version, by just looking at the jar names listed in systemtest.getDependency build https://hyc-runtimes-jenkins.swg-devops.com/job/systemtest.getDependency/. 

However, since ASM jar needs updates more regularly than most other jars, we could have the generic name for just the ASM jar.